modules/installation-aws-config-yaml-customizations.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/ipi/installing-aws-customizations.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="installation-aws-config-yaml-customizations_{context}"]
+= Sample customized install-config.yaml file for AWS
+
+You can customize the installation configuration file (`install-config.yaml`) to specify more details about your {product-title} cluster's platform or modify the values of the required parameters.
+
+[IMPORTANT]
+====
+This sample YAML file is provided for reference only. You must obtain your
+`install-config.yaml` file by using the installation program and modify it.
+====
+
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: example.com
+credentialsMode: Mint
+controlPlane:
+  hyperthreading: Enabled
+  name: master
+  platform:
+    aws:
+      zones:
+      - us-west-2a
+      - us-west-2b
+      rootVolume:
+        iops: 4000
+        size: 500
+        type: io1
+      metadataService:
+        authentication: Optional
+      type: m6i.xlarge
+  replicas: 3
+compute:
+- hyperthreading: Enabled
+  name: worker
+  platform:
+    aws:
+      rootVolume:
+        iops: 2000
+        size: 500
+        type: io1
+      metadataService:
+        authentication: Optional
+      type: c5.4xlarge
+      zones:
+      - us-west-2c
+  replicas: 3
+metadata:
+  name: test-cluster
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: 10.0.0.0/16
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  aws:
+    region: us-west-2
+    propagateUserTags: true
+    userTags:
+      adminContact: jdoe
+      costCenter: 7536
+    amiID: ami-0c5d3e03c0ab9b19a
+    serviceEndpoints:
+      - name: ec2
+        url: https://vpce-id.ec2.us-west-2.vpce.amazonaws.com
+fips: false
+sshKey: ssh-ed25519 AAAA...
+pullSecret: '{"auths": ...}'
```

Comment:
```suggestion
apiVersion: v1
baseDomain: example.com
controlPlane:
  name: master
  platform:
    aws:
      zones:
      - us-west-2a
      - us-west-2b
      rootVolume:
        iops: 4000
        size: 500
        type: io1
      type: m6i.xlarge
  replicas: 3
compute:
-  name: worker
  platform:
    aws:
      rootVolume:
        iops: 2000
        size: 500
        type: io1
      type: c5.4xlarge
      zones:
      - us-west-2c
  replicas: 3
metadata:
  name: test-cluster
platform:
  aws:
    region: us-west-2
    propagateUserTags: true
    userTags:
      adminContact: jdoe
      costCenter: 7536
sshKey: ssh-ed25519 AAAA...
pullSecret: '{"auths": ...}'
```

In case this is hard to read, I would drop:
credentialsMode: Mint
  hyperthreading: Enabled
  metadataService:
        authentication: Optional
all of networking
    amiID: ami-0c5d3e03c0ab9b19a
serviceEndpoints:
      - name: ec2
        url: https://vpce-id.ec2.us-west-2.vpce.amazonaws.com
fips: false

It kind of depends on how aggressive you want to be in terms of paring down the install config. Everything I removed seems very unlikely to be changed--except `credentialsMode` but I think that is documented elsewhere.

Most of what I left is related to the cloud infrastructure and these are things I could see customers wanting to tweak. We could probably remove `zones` as well, as I'm not sure how often users want to choose specific zones. The installer just distributes the machine pools over those zones.